tests/sentry/api/endpoints/test_project_metrics_extraction_rules.py

```diff
@@ -4,7 +4,10 @@
 
```

Comment:
Did you do some due-diligence concerning the permission levels of 
1. deletion of extraction rules
2. delection of alert rules
3. deletion of dashboard widgets

Do they all have the same permissions, or could some privilege escalation happen here somehow? (e.g. user being allowed to delete the extraction rule, but not alertrules)